client/src/constants/routes.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-useless-escape */
 export default Object.freeze({
   // Hostname
-  ParticipantHostname: 'hcapparticipants.*',
-  EmployerHostname: 'hcapemployers.*',
+  ParticipantHostname: RegExp('^(www\.)?hcapparticipants\..*'),
+  EmployerHostname: RegExp('^(www\.)?hcapemployers\..*'),
```

Comment:
As it stands, these urls contain a wildcard match at the end, meaning they are valid for any domain provided it starts in the right way? Is that a security issue or are we not concerned with that at this level?